packages/components/src/templates/notification.stories.ts

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import '../solid-components';
+import { html } from 'lit';
+
+export default {
+  title: 'Templates/Notification',
+  tags: ['!dev'],
+  parameters: {
+    chromatic: { disableSnapshot: true }
+  }
+};
+
+export const InfoNotification = {
+  name: 'Info Notification',
+  render: (_args: Record<string, any>) => {
+    return html`
+      <style>
+        sd-notification.info-variant::part(message) {
+          display: flex;
+          justify-content: space-between;
+          align-items: center;
+        }
+      </style>
+
+      <sd-notification class="info-variant" variant="info" open>
+        Editing is restricted
+        <sd-button variant="secondary" class="mr-4">Request to edit</sd-button>
+      </sd-notification>
+    `;
+  }
+};
+
+export const InfoToastNotification = {
```

Comment:
That’s not working inside the iframe I guess ... and the padding left & right for button/secondary are incorrect ...

Reply:
Now the toast pops up. THX

But:

padding left & right for button/secondary are incorrect:
<img width="282" alt="Bildschirmfoto 2024-09-03 um 12 53 09" src="https://github.com/user-attachments/assets/ac03706a-dc22-4f95-93b6-2e25f0851b86">
Should be 16px

Reply:
Updated